lib/src/tag_detection.dart

```diff
@@ -806,15 +813,24 @@ class Tagger {
             .toList()
         : <String>[];
     final isBinaryOnly = nonSrcDartFiles.isEmpty && allBinFiles.isNotEmpty;
+    final isAssetOnly = libDartFiles.isEmpty && allBinFiles.isEmpty;
 
     return Tagger._(
-        pubspec.name, session, pubspecCache, isBinaryOnly, topLibraries);
+      pubspec.name,
+      session,
+      pubspecCache,
+      isAssetOnly,
+      isBinaryOnly,
+      topLibraries,
+    );
   }
 
   void sdkTags(List<String> tags, List<Explanation> explanations) {
     try {
-      if (_isBinaryOnly) {
-        tags.add('sdk:dart');
+      if (_isAssetOnly) {
+        tags.addAll(_Sdk.knownSdks.map((s) => s.tag));
```

Comment:
What if the package has an SDK constraint on `flutter`, or a dependency on something else that has an SDK constraint on flutter..

I agree, we don't need to look at the file in the dependenices because those are not imported.

Reply:
Good point, we'll need to run the declared platform and sdk finders to only surface the ones that are declared.

Reply:
Fixed. PTAL.